### PR TITLE
Centcomm orders

### PR DESCRIPTION
**Pull request by MadmanMartian**
It's been about 2 years since I mucked up salvaging centcomm orders, this should be a better system.

 - Material worth moved to defines
 - Removes the switch statement that was the previous centcomm order, moves them to datum defines so it's easier to make new ones
 - 3 Centcomm orders are generated at roundstart for cargo to pursue

:cl:
 * rscadd: 3 Centcomm orders are now generated at roundstart

### CODE REVIEW

**Comment by SGuivenne** [comment]
Please, as captain I'm always hoping to get tasks from Centcomm (read: admins) because otherwise its just "convince the station to run, what is profit" and that bores me slightly.

**Comment by MadmanMartian** [comment]
Is that a positive or a negative response to the PR?

**Comment by Goldsteel** [comment]
I think this is a positive thing.

File: code/game/centcomm_orders.dm [hunk 1]
```diff
@@ -131,4 +132,170 @@ var/global/current_centcomm_order_id=124901
...
+	var/list/orders = list()
+	orders.Add(subtypesof(/datum/centcomm_order/department/cargo))
+	orders.Add(subtypesof(/datum/centcomm_order/department/science))
+	orders.Add(subtypesof(/datum/centcomm_order/department/medical))
+	orders.Add(subtypesof(/datum/centcomm_order/department/civilian))
+
+	return orders
+
+/proc/create_random_order()
+	var/choice = pick(get_potential_orders())
```
Comment (Kurfursten) [inline code/game/centcomm_orders.dm]: pick_n_take might feel more fair

**Pull request merged**